installation-scripts-onm/MASTER_START_SCRIPT.sh

```diff
@@ -108,4 +108,16 @@ if [ "$SERVERLESS_ENABLED" == "yes" ]; then
 
   kubectl apply -f https://raw.githubusercontent.com/kubevela/samples/master/06.Knative_App/componentdefinition-knative-serving.yaml
   echo "Serverless installation completed."
+fi
+
+if [ "$WORKFLOW_ENABLED" == "yes" ]; then
+  echo "Workflow installation.";
+
+  helm install argo-workflows argo-workflows --repo https://argoproj.github.io/argo-helm --namespace argo --create-namespace --values workflow/workflow.yaml;
```

Comment:
AFAIU, SAL does not see/transfer these files so your best bet for now is a workaround like above on L109.

Reply:
I've just added the wget file fetching like above.

Reply:
Thanks, LGTM.